elements/lisk-p2p/src/p2p_types.ts

```diff
@@ -35,51 +37,87 @@ export interface P2PPenalty {
 	readonly penalty: number;
 }
 
+export interface P2PSharedState {
+	readonly height: number;
+	readonly updatedAt?: Date;
+	readonly os?: string;
+	readonly version: string;
+	readonly protocolVersion: string;
+	// tslint:disable-next-line: no-mixed-interface
+	readonly [key: string]: unknown;
+}
+
+export interface P2PInternalState {
+	readonly dateAdded?: Date;
```

Comment:
This parameter looks like overlapping with https://github.com/LiskHQ/lisk-sdk/blob/a32eb03c8d0aa8836ca00f3d1089e34d845cc515/elements/lisk-p2p/src/peer_book/base_list.ts#L31 Shall we remove one of them? Shall we remove `CustomPeerInfo` interface and reuse the new `P2PPeerInfo`?

Reply:
In future we might have more values that we need within peer bucket so its better to still keep it and also `dateAdded` is removed from P2PPeerInfo and its a required field in bucket and also triedPeers use `maxConnectRetries`